packages/axway-cli-pm/src/utils.js

```diff
@@ -49,7 +49,6 @@ export class ListrTextRenderer {
 export async function runListr({ console, json, tasks }) {
 	await (new Listr(tasks, {
 		concurrent: 10,
-		console,
```

Comment:
Ha, good catch! `console` was never used!

Reply:
Oh hold on. It looks like `console` was intended to be used by your custom `ListrTextRenderer` type, right?

Reply:
I re-adding `console` arg since custom `ListrTextRenderer` needs it.